src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -69,6 +78,7 @@ public Command parseCommand(String userInput) throws ParseException {
             return new HelpCommand();
 
         default:
+            logger.finer("ParseException: " + userInput);
```

Comment:
Suppose the user input `blah` caused a ParseException. Compare the following two log messages:

`ParseException: blah`
`This user input cased a ParseException: blah`

I think the latter is more self-explanatory. The former can even mislead the reader to think `blah` _is_ the ParseException. 
What do you think?